GetARock/GetARock/Global/Extension/String+Extension.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  String+Extension.swift
+//  GetARock
+//
+//  Created by Jisu Jang on 2023/02/01.
+//
+
+import Foundation
+
+//MARK: 특수문자 제한 정규표현식
+
+extension String {
+    func checkStringExpression() -> Bool{
+           do {
+               let regex = try NSRegularExpression(
+                pattern: "^[a-zA-Z0-9가-힣ㄱ-ㅎㅏ-ㅣ]$",
```

Comment:
p1;

피그마 확인하니까 공백은 입력 가능한 것 같아요!
아래처럼 \\\s 추가하면 될 것 같습니다
```suggestion
                pattern: "^[a-zA-Z0-9가-힣ㄱ-ㅎㅏ-ㅣ\\s]$",
```

Reply:
오옷 감사합니다! 적용했어요~  https://github.com/extreme-rock/GetARock-iOS/pull/71/commits/5467bfb769474b9e110024c7f957d92401deb56c